docs/audit.md

```diff
@@ -411,6 +415,10 @@ Failed conditions:
 
                 Run detect-secrets audit .secrets.baseline, and audit all potential secrets.
 
+                It is recommended to configure the pre-commit hook for your project: https://github.com/IBM/detect-secrets/blob/master/docs/developer-tool-faqmd#how-do-i-set-up-the-pre-commit-hook - it automatically scans your code for secrets and blocks local commits if potential secrets are found.
+
+                The --fail-on-unaudited option can be added to your pre-commit config file to prevent unaudited secrets from being committed locally: https://github.com/IBM/detect-secrets/blob/master/docs/cheat-sheetmd#fail-pre-commit-if-there-are-non-audited-entries
```

Comment:
#fail-pre-commit-if-there-are-non-audited-entries  <--- In m head I'm think should be #fail-pre-commit-if-there-are-unaudited-entries

Reply:
maybe I a wrong ---> In general, search you code and readme for the word "non-audited" and make sure you meant to leave it there or have "unaudited" -- as I see "non-audited" in a couple places

Reply:
thanks for double-checking this, I've updated the `non-audited` references here: https://github.com/IBM/detect-secrets/pull/91/commits/282107f41f27d9585fa2907f044e53baa0cfb1d7.